### ISSUE TITLE
Indentation-related parsing bug in raw HTML inside fenced div

### ISSUE THREAD

**Issue by jgm** [jgm/pandoc#5360]
Consider
```
::: {.output}
<table>
 <tr>
  <td>hi</td>
 </tr>
</table>
:::
```
This does not get parsed as a fenced div, but if we remove one space it works:
```
::: {.output}
<table>
 <tr>
  <td>hi</td>
</tr>
</table>
:::
```
Originally noted in #5354.

**Comment by jgm** [comment]
See the issue5354 branch for a test case and some attempts.

**Comment by tarleb** [comment]
Closed by 82f44592a.
